BEAMS3D/Sources/beams3d_physics_mod.f90

```diff
@@ -7,2058 +7,2099 @@
+      NION, NI_AUX_M, NI_AUX_Z, vll_start, charge_beams
+   USE beams3d_lines, ONLY: R_lines, Z_lines, PHI_lines, &
+      vr_lines, vphi_lines, vz_lines, &
+      myline, mytdex, moment, ltherm, &
+      nparticles, vll_lines, &
+      moment_lines, mybeam, mycharge, myZ, &
+      mymass, myv_neut, rand_prob, &
+      cum_prob, tau, &
+      epower_prof, ipower_prof, &
+      end_state, fact_crit, fact_pa, &
+      fact_vsound, fact_coul, fact_kick, &
+      ns_prof1, ns_prof2, ns_prof3, ns_prof4, &
+      ns_prof5, my_end
+   USE beams3d_grid, ONLY: BR_spl, BZ_spl, delta_t, BPHI_spl, &
+      MODB_spl, MODB4D, &
+      phimax, S4D, X4D, Y4D, TE4D, NE4D, TI4D, ZEFF4D, &
+      nr, nphi, nz, rmax, rmin, zmax, zmin, &
+      phimin, eps1, eps2, eps3, raxis, phiaxis,&
+      zaxis, U4D,nzeff, &
+      hr, hp, hz, hri, hpi, hzi, &
+      B_kick_min, B_kick_max, E_kick, freq_kick, &
+      plasma_mass, NI5D, BR4D, BZ4D, BPHI4D
+   USE EZspline_obj
+   USE EZspline
+   USE adas_mod_parallel
+   USE mpi_params
+
+   !-----------------------------------------------------------------
+   !     Module PARAMETERS
+   !-----------------------------------------------------------------
+
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: electron_mass = 9.10938356D-31 !m_e
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: e_charge      = 1.60217662E-19 !e_c
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: sqrt_pi       = 1.7724538509   !pi^(1/2)
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: inv_sqrt2     = 0.7071067812   !1/sqrt(2)
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: mpome         = 5.44602984424355D-4 !e_c
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: inv_dalton    = 6.02214076208E+26 ! 1./AMU [1/kg]
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: inv_Ae    = 1.0/1836.1d0 ! AMU./electron_mass
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: inv_cspeed    = 3.3356409520E-09 ! 1./c [s/m]
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: zero          = 0.0D0 ! 0.0
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: half          = 0.5D0 ! 1/2
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: one           = 1.0D0 ! 1.0
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: eps_0 = 8.854187817E-12;
+   DOUBLE PRECISION, PRIVATE, PARAMETER :: hbar = 1.054571817E-34;
+
+   !-----------------------------------------------------------------
+   !     SUBROUTINES
+   !        BEAMS3D_PHYISCS(t,q) Slowing down and pitch angle
+   !        BEAMS3D_FOLLOW_NEUT(t,q) Neutral particle following
+   !-----------------------------------------------------------------
+CONTAINS
+
+   !-----------------------------------------------------------------
+   !     Function:      coulomb_log_nrl19
+   !     Authors:       S. Lazerson (samuel.lazerson@ipp.mpg.de)
+   !     Date:          07/05/2023
+   !     Description:   Coulomb log as defined in NRL 2019
+   !-----------------------------------------------------------------
+   FUNCTION coulomb_log_nrl19(ne_in,te_in,vbeta_in,Zeff_in)  result(slow_par)
```

Comment:
What is the purpose of these changes? Also the documentation should be updated so that the function outputs are clear.